packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -73,7 +73,7 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
       imageUrl={data?.imageUrl ?? props.imageUrl}
       imageInitials={data?.imageInitials ?? props.imageInitials}
       initialsColor={data?.initialsColor ?? props.initialsColor}
-      initialsTextColor={data?.initialsTextColor ?? props.initialsTextColor}
+      initialsTextColor={data?.initialsTextColor ?? props.initialsTextColor ?? 'white'}
```

Comment:
In the doc comments for this prop add `@defaultValue white` to this one

Reply:
done ;)